### PR TITLE
Feature/fixes

### PR DESCRIPTION
**Pull request by Ezeh20**
- corrected  question number 2 for the course category 
- Increased the size of the select option
- reverted to the native date picker  

Screenshot (110)
Screenshot (111)
Screenshot (112)

### CODE REVIEW

File: src/sections/NotifyMe/internals/NotifyModal/NotifyModal.jsx [hunk 1]
```diff
@@ -1,15 +1,11 @@
-import React, { useEffect } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import { ModalContext } from '../../../../context/ModalContext';
 import { useContext } from 'react';
 
 export const NotifyModal = ({ children, className }) => {
   const { notifyPopModal } = useContext(ModalContext);
 
-  useEffect(() => {
```
Comment (edustipend-root) [inline src/sections/NotifyMe/internals/NotifyModal/NotifyModal.jsx:9]: Why did we remove this code? @Ezeh20

**Review by edustipend-root — COMMENTED** [review]
@Ezeh20, can you add a screenshot of what the UI looks like after the date picker change?

File: src/sections/NotifyMe/internals/NotifyModal/NotifyModal.jsx (hunk 1 above)
Comment (Ezeh20) [reply src/sections/NotifyMe/internals/NotifyModal/NotifyModal.jsx:9]: it's causes some pages to have an overflow of hidden, you can't really scroll till you refresh

**Comment by Ezeh20** [comment]
Screenshot (111)

**Pull request merged**